index.bs

```diff
@@ -265,11 +265,26 @@ although they may be harder to find.
 Simpler features are easier for user agents to implement and test,
 more likely to be interoperable,
 and easier for authors to understand.
+It is especially important to design your feature so that 
+the most common use cases are easy to accomplish.
 
 Make sure that your <a href="#priority-of-constituencies">user needs</a>
 are well-defined.
 This allows you to avoid scope creep,
 and make sure that your API does actually meet the needs of all users.
+Of course,
+complex or rare use cases are also worth solving,
+though their solutions may be more complicated to use.
+As Alan Kay said,
+"simple things should be simple, complex things should be possible."
+
+Do note however that while common cases are often simple,
+commonality and complexity are not always correlated.
+
+<div class=example>
+Sanitizing HTML to prevent XSS attacks is a complex process that requires extensive security knowledge, however the Sanitizer API provides a shortcut for this common use case. 
```

Comment:
```suggestion
Sanitizing HTML to prevent XSS attacks is a complex process
that requires extensive security knowledge,
however the Sanitizer API provides a shortcut for this common use case. 
```